### DISCUSSION TITLE
What 🇻🇦 doctrine has been denounced recently?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#884] [Galaxy Trivia] [answered]
**Bonus:** How many years did it last?

**Comment by peterbaccaro** [comment, marked as answer]
The Vatican recently denounced the “**Doctrine of Discovery**”, a 15th-century concept laid out in so-called “papal bulls” that were used to justify European Christian colonialists’ seizure of Indigenous lands in Africa and the Americas.

The doctrine was laid out in a series of papal "bulls," or decrees; the first one was issued in 1452.  Therefore the Doctrine of Discovery had lasted around **571 years**.
